### DISCUSSION TITLE
Switch SuperAdmin certificate so I can delete the old CA

### DISCUSSION THREAD

**Discussion by xxedgexx** [Keyfactor/ejbca-ce#265] [General]
I created a new CA, but the SuperAdmin role is using the old CA.  It won't allow me to remove it because I'm logged in using that existing SuperAdmin certificate.

How do I go about regenerating the SuperAdmin role under the new CA so I can delete the old one?

Thanks
-jeremy

**Comment by mike-agrenius-kushner** [comment]
Hi, assuming your old one still has access to "/", then you can just generate a new end entity and add that certificate to the same Role. 

If not, you can use the EJBCA CLI from the console to generate a new admin.

Cheers,
Mike
